src/Controller/Admin/Asset/AssetHelperController.php

```diff
@@ -636,7 +636,7 @@ protected function updateGridConfigFavourites(?GridConfig $gridConfig, array $me
                     }
 
                     // Check if the user is the owner. If that is the case we do not update the favourite
-                    if ((int) $favouriteGridConfig->getOwnerId() === (int) $id) {
+                    if ((int) $favouriteGridConfig->getOwnerId() === $id) {
```

Comment:
getOwnerId() can only be int or null. So there is also no convert needed.
```suggestion
                    if ($favouriteGridConfig->getOwnerId() === $id) {
```